queue_job/tests/test_json_field.py

```diff
@@ -14,8 +14,24 @@
 
 
 class TestJson(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        User = cls.env["res.users"]
+        main_company = cls.env.ref("base.main_company")
+        group_user = cls.env.ref("base.group_user")
+        cls.demo_user = User.create(
```

Comment:
```suggestion
        main_company = cls.env.ref("base.main_company")
        group_user = cls.env.ref("base.group_user")
        cls.demo_user = cls.env["res.users"].create(
```
A bit opnionated, but the `User` variable is unnecessary and Camel cased variables should not be used.

Reply:
Camelcase is ok in this case as `cls.env["res.users"]` is (roughly) a class.